src/storage/storage-types.ts

```diff
@@ -138,41 +186,3 @@ export interface IStorageDriverAsync extends IStorageAsyncConfig {
     // return a copy of the doc, frozen, with _localIndex set.
     upsert(doc: Doc): Promise<Doc>;
 }
-
-//================================================================================ 
-// EVENTS
-
-export enum IngestResult {
-    // doc was not saved: negative numbers
-    WriteError = 'WRITE_ERROR',
-    ObsoleteFromSameAuthor = 'OBSOLETE_FROM_SAME_AUTHOR',
-    AlreadyHadIt = 'ALREADY_HAD_IT',
-    Invalid = 'INVALID_DOCUMENT',
-
-    // doc was saved: positive numbers
-    AcceptedButNotLatest = 'ACCEPTED_BUT_NOT_LATEST',
-    AcceptedAndLatest = 'ACCEPTED_AND_LATEST',
-}
```

Comment:
`IngestResult` is gone now, replaced with `IngestEvent` which has richer context information.